onnx/backend/test/case/model/update.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+from typing import Sequence
+
+
+class Update(Base):
+
+    @staticmethod
+    def export():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Add', ['x', 'y'], ['z'], name='MyAdd')
+        updateNode = onnx.helper.make_node(
+            'Update', ['z'], ['x'], name='MyUpdate')
```

Comment:
Sorry, this example seems to violate several fundamental principles of ONNX. Unclear if 'x' is an input or output to Update (the above seems to conflict with the opschema). In either case, we have problems. May be it will be easier if we discuss this face to face.